x/supplier/types/keys.go

```diff
@@ -14,6 +18,15 @@ const (
 	MemStoreKey = "mem_supplier"
 )
 
-func KeyPrefix(p string) []byte {
-	return []byte(p)
+// KeyDelimiter is the delimiter for composite keys.
+var KeyDelimiter = []byte("/")
+
+// KeyPrefix returns the given prefix as a byte slice for use with the KVStore.
+func KeyPrefix(prefix string) []byte {
```

Comment:
NIT: Consider whether or not this needs to be exported.

Reply:
Basically everything in the `types` pkg needs to be exported because by definition, it consists of things that are shared dependencies of multiple pkgs.